lib/clients/influxdb.js

```diff
@@ -58,12 +58,13 @@ InfluxUdp.prototype.buildMessages = function(data) {
   });
   var message = [keys.join(','), fields.join(',')];
   if (data.timestamp) {
-    message.push(data.timestamp);
+    message.push(data.timestamp*1e6); //it has to be nano seconds
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ATrailingComment)